From 59ba10cd69b8cf103922b7e9dfff61b0bd846c3c Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Mon, 16 Mar 2020 21:37:50 +0100
Subject: [PATCH 1/1] xlat: add IPPROTO_ETHERNET to inet_protocols

* xlat/inet_protocols.in (IPPROTO_ETHERNET): New constant, introduced
by Linux commit v5.6-rc6~23^2~9.

Signed-off-by: Andrei Gherzan <andrei.gherzan@huawei.com>
Upstream-status: Backport
---
 xlat/inet_protocols.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xlat/inet_protocols.in b/xlat/inet_protocols.in
index 5c27821a3..c37508fa3 100644
--- a/xlat/inet_protocols.in
+++ b/xlat/inet_protocols.in
@@ -30,6 +30,7 @@ IPPROTO_SCTP		132
 IPPROTO_MH		135
 IPPROTO_UDPLITE		136
 IPPROTO_MPLS		137
+IPPROTO_ETHERNET	143
 IPPROTO_RAW		255
 IPPROTO_MAX		256
 IPPROTO_MPTCP		262
-- 
2.33.0

